guide.md

```diff
@@ -30,9 +30,9 @@ The manual is here: <https://briarproject.org/manual/>, quick-start guide here:
 
 This guide is a non-profit open-source initiative, licensed under Creative Commons **Attribution-NonCommercial** 4.0 International ([cc-by-nc-4.0][] <sup>[[Archive.org]][27]</sup>).
 
--   For mirrors see [Appendix A6: Mirrors]
+-   For mirrors see [Appendix A6: Mirrors:]

```

Comment:
Keep the : in the ToC items but remove them for the links in the body? That works too. It would look like:
`[Here's an example of TRIM on Windows][Here's an example of TRIM on Windows:].`